.travis.yml

```diff
@@ -50,12 +51,12 @@ jobs:
     #  script: npm run test
 
     # Run tests on multiple browsers and publish code coverage
-    - stage: test
-      name: Cross-Browser Tests
-      script:
-        - npm run browser-test
-        - npm run coverage
-        - codecov
+    # - stage: test
```

Comment:
whats the deal with these?

Reply:
The airtap tests are currently broken due to weird sourcemap problems with multiple entry paths, so `npm run test:interactive` is the only working test suite.

Reply:
sg, we gonna remove thse or fix later?

Reply:
Still undecided. My current tests look nice, but they don't integrate with CI so the test suite will eventually need to be updated.

Reply:
Obviously not ideal to have manual tests but it’s ok with me since the assumption is that this module won’t be updating that frequently. If we want to one day make them ci friendly we can open an issue